<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartWindow</class>
 <widget class="QWidget" name="ChartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chart Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QtCharts::QChartView" name="chartView_1"/>
   </item>
   <item row="0" column="1">
    <widget class="QtCharts::QChartView" name="chartView_2"/>
   </item>
   <item row="1" column="0">
    <widget class="QtCharts::QChartView" name="chartView_3"/>
   </item>
   <item row="1" column="1">
    <widget class="QtCharts::QChartView" name="chartView_4"/>
   </item>
   <item row="2" column="0">
    <widget class="QtCharts::QChartView" name="chartView_5"/>
   </item>
   <item row="2" column="1">
    <widget class="QtCharts::QChartView" name="chartView_6"/>
   </item>
   <item row="3" column="0">
    <widget class="QtCharts::QChartView" name="chartView_7"/>
   </item>
   <item row="3" column="1">
    <widget class="QtCharts::QChartView" name="chartView_8"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts/QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>

